docs/components/InputFieldWrapper/InputFieldWrapper.stories.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Components/Private/InputFieldWrapper/Docs" />
+
+# Input Field Wrapper (Internal)
+
+This component is used to wrap input fields such as `InputText` and
+`InputPressable` to provide common patterns and functionality present in our
+input designs, such as: suffix/prefix icons, placeholders, etc.
+
+Note: example does not have a child component, so no interaction will occur on
```

Comment:
I don't think you need this here anymore.  The example is in the Mobile folder and it has the disclaimer on it.